src/components/ModelingSidebar/ModelingSidebar.tsx

```diff
@@ -79,7 +79,7 @@ export function ModelingSidebar({ paneOpacity }: ModelingSidebarProps) {
       title: 'Load external model',
       sidebarName: 'Load external model',
       icon: 'importFile',
-      keybinding: 'Ctrl + Shift + I',
+      keybinding: 'Mod + Alt + L',
```

Comment:
Also fixing this quick while we're here. Ctrl + Shift + I was used for dev tools on some platforms already.